pisek/jobs/jobs.py

```diff
@@ -364,31 +372,20 @@ def _job_states(self) -> tuple[State, ...]:
         """States of this manager's jobs."""
         return tuple(map(lambda j: j.state, self.jobs))
 
-    def _jobs_with_state(self, state: State) -> list[Job]:
+    def jobs_with_state(self, state: State) -> list[Job]:
         """Filter this manager's jobs by state."""
         return list(filter(lambda j: j.state == state, self.jobs))
 
     def _update(self) -> None:
         """Override this function for manager-specific."""
         pass
 
-    def update(self) -> str:
+    def update(self) -> None:
         """Update this manager's state according to its jobs and return status."""
```

Comment:
```suggestion
        """Update this manager's state according to its jobs."""
```

Is the first part still true?